dev/scripts/src/alluxio.org/build-distribution/cmd/root.go

```diff
@@ -48,4 +49,7 @@ func additionalFlags(cmd *flag.FlagSet) {
 	cmd.BoolVar(&debugFlag, "debug", false, "whether to run this tool in debug mode to generate additional console output")
 	cmd.StringVar(&ufsModulesFlag, "ufs-modules", strings.Join(defaultModules(ufsModules), ","),
 		fmt.Sprintf("a comma-separated list of ufs modules to compile into the distribution tarball(s). Specify 'all' to build all ufs modules. Supported ufs modules: [%v]", strings.Join(validModules(ufsModules), ",")))
+	cmd.StringVar(&includedLibJarsFlag, "lib-jars", "all",
+		"a comma-separated list of jars under lib/ needed to included except underfs-hdfs modules. All jars under lib/ will be included by default."+
```

Comment:
slight rewording of the flag description:
`a comma-separted list of jars under lib/ to include in addition to all underfs-hdfs modules. All jars under lib/ will be included by default.`

also move this flag declaration up a line for alphabetical order in declaration of flags